schemata/export_files_check_batch.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "http://json-schema.org/draft-04/hyper-schema",
+  "id": "/schemata/export_files_check_batch",
+
+  "title": "Export/Report Files (check status in batch)",
+  "stability": "production",
+  "description": "Closely mimics the \"Recent Files\" screen in the WebAdMIT application.  This call will display all export files that have recently run under a particular identity, and their status.  Importantly, the download URL is not displayed in this response as these individual expiring URLs can only be generated by calling for the status of an individual file directly using the previous call.  You can use the results of this call and loop through each one to get a download URL for each export file marked as \"Available\".",
+
+  "links": [
+    {
+      "description": "The response is the collection of id’s and statuses of the running exports for a specific identity within a user’s WebAdMIT profile.",
```

Comment:
I know this matches the PDF, but apostrophes aren't used for plurals.  😢   (`id's`)